README.md

```diff
@@ -1,2 +1,211 @@
 # aurelia-config
-Configuration plugin that allows you to expose your configuration.
+
+Aurelia-config is configuration loader and handler (using [homefront](https://www.npmjs.com/package/homefront)) that allows you to expose and centralize your configuration.
```

Comment:
I'd like the readme to be a bit more explanatory, or even better: have a book. The usage is a bit obscure imo. How do I actually use this? 

> a local configuration which which exports a object named 'config'

Except, it doesn't. It exports an object called `appConfig`. So does it then not matter?

I'll add more inline feedback.